configs/BOTWINGF405/config.h

```diff
@@ -0,0 +1,130 @@
+#define BEEPER_PIN                          PB0
+
+#define MOTOR1_PIN                          PA9
+#define MOTOR2_PIN                          PA8
+#define MOTOR3_PIN                          PC9
+#define MOTOR4_PIN                          PC8
+
+#define LED_STRIP_PIN                       PB3
+
+#define UART1_TX_PIN                        PB6
+#define UART1_RX_PIN                        PB7
+
+#define UART2_TX_PIN                        PA2
+#define UART2_RX_PIN                        PA3
+
+#define UART3_TX_PIN                        PB10
+#define UART3_RX_PIN                        PB11
+
+#define UART4_TX_PIN                        PA0
+#define UART4_RX_PIN                        PA1
+
+#define UART5_TX_PIN                        PC12
+#define UART5_RX_PIN                        PD2
+
+#define UART6_TX_PIN                        PC6
+#define UART6_RX_PIN                        PC7
+
+#define I2C1_SCL_PIN                        PB8
+#define I2C1_SDA_PIN                        PB9
+
+#define LED0_PIN                            PC0
+#define LED1_PIN                            PC5
+
+#define SPI1_SCK_PIN                        PA5
+#define SPI1_SDI_PIN                        PA6
+#define SPI1_SDO_PIN                        PA7
+
+#define SPI2_SCK_PIN                        PB13
+#define SPI2_SDI_PIN                        PB14
+#define SPI2_SDO_PIN                        PB15
+
+#define SPI3_SCK_PIN                        PC10
+#define SPI3_SDI_PIN                        PC11
+#define SPI3_SDO_PIN                        PB5
+
+#define ADC_VBAT_PIN                        PC1
+#define ADC_RSSI_PIN                        PC2
+#define ADC_CURR_PIN                        PC3
+
+#define PINIO1_PIN                          PB1
+
+#define SWDIO_PIN                           PA13
+#define SWCLK_PIN                           PA14
+
+#define TIMER_PIN_MAPPING \
+                                            TIMER_PIN_MAP(0, PB3, 1, 0) \
+                                            TIMER_PIN_MAP(1, PA9, 1, 0) \
+                                            TIMER_PIN_MAP(2, PA8, 1, 0) \
+                                            TIMER_PIN_MAP(3, PC9, 1, 1) \
+                                            TIMER_PIN_MAP(4, PC8, 1, 0) 
+
+#define ADC1_DMA_OPT                        0
+#define ADC_INSTANCE                        ADC1
+
+#define DEFAULT_BLACKBOX_DEVICE             BLACKBOX_DEVICE_FLASH
+
+#define DEFAULT_CURRENT_METER_SOURCE        CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE        VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE         400
+#define DEFAULT_VOLTAGE_METER_SCALE         110
+
+#define PINIO1_BOX                          40
+
+#define SERIALRX_UART                       SERIAL_PORT_USART1
+#define ESC_SENSOR_UART                     SERIAL_PORT_USART2
+#define GPS_UART                            SERIAL_PORT_USART6
+
+#define DEFAULT_FEATURES                    (FEATURE_OSD | FEATURE_LED_STRIP)
```

Comment:
Why are these defined in the config? Are any included on the FC?

Reply:
Removed :+1: